deployer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nimbella-deployer",
-  "version": "1.6.1",
+  "version": "1.6.2",
```

Comment:
As discussed, don't increment versions in a PR.   That is always handled as part of the release process.